### PR TITLE
create text_2 file.

### PR DESCRIPTION
**Pull request by NoCh-Git**
Please review.

### CODE REVIEW

**Comment by Toukant** [comment]
Hello

**Review by Toukant — COMMENTED** [review]
hi!

File: test_2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+There is some content here. And here! And here!
```
Comment (Toukant) [inline test_2.txt:1]: And here as well!

Comment (NoCh-Git) [reply test_2.txt:1]: Ah I see!

**Pull request merged**